requirements.txt

```diff
@@ -1,9 +1,9 @@
-boto==2.48.0
-coverage==4.2
+boto==2.49.0
+coverage==5.0.3
 funcsigs==1.0.2
-mock==2.0.0
-pbr==1.10.0
-py==1.4.31
-pylint==1.7.4
-pytest==3.0.3
-six==1.10.0
+mock==3.0.5
+pbr==5.4.4
+py==1.8.1
+pylint==2.4.4
+pytest==5.3.2
+six==1.13.0
```

Comment:
still needed?

Reply:
six? good point. I'll remove it and see what happens

Reply:
done